libraries/Microsoft.Bot.Builder.AI.QnA/Utils/QnACardBuilder.cs

```diff
@@ -77,48 +77,82 @@ public static IMessageActivity GetSuggestionsCard(List<string> suggestionsList,
         }
 
         /// <summary>
-        /// Get active learning suggestions card.
+        /// Get Card for MultiTurn scenario. (Can be deprected from 4.10.0 release of sdk).
         /// </summary>
         /// <param name="result">Result to be dispalyed as prompts.</param>
-        /// <param name="cardNoMatchText">No match text.</param>
+        /// <param name="cardNoMatchText">No match text.</param>       
         /// <returns>IMessageActivity.</returns>
         public static IMessageActivity GetQnAPromptsCard(QueryResult result, string cardNoMatchText)
+        {
+            return GetQnADefaultResponse(result, true);
+        }
+
+        /// <summary>
+        /// Get Card for Default QnA Maker scenario.
+        /// </summary>
+        /// <param name="result">Result to be dispalyed as prompts.</param>
+        /// <param name="displayPreciseAnswerOnly">renderingchoice.</param>
```

Comment:
What does this comment mean?

Reply:
Updated content.

---
In reply to: [430559229](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430559229) [](ancestors = 430559229)